src/shim.node.ts

```diff
@@ -10,3 +10,14 @@ export const Deno = {
   env: { get: (key: string) => process.env[key] },
   exit: process.exit,
 };
+
+export async function fetch(
+  localPath: string,
+  // deno-lint-ignore no-explicit-any
+): Promise<{ json: () => Promise<any> }> {
+  if (!localPath.startsWith("file:")) {
+    throw new Error("Can only read local files!");
+  }
+  const data = await readFile(localPath.substr(5), { encoding: "utf-8" });
```

Comment:
Slighly neater with `URL`:

```suggestion
  url: URL,
  // deno-lint-ignore no-explicit-any
): Promise<{ json: () => Promise<any> }> {
  if (url.protocol !== "file:") {
    throw new Error("Can only read local files!");
  }
  const data = await readFile(url, { encoding: "utf-8" });
```